_includes/touchpoint-survey-script.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://touchpoints.app.cloud.gov/touchpoints/5c2410e4.js" async></script>
+
+<section class="touchpoints__container" role="complementary" aria-label="page feedback">
+    <a href="#" class="usa-button usa-button--outline" id="#touchpoints-modal-button" aria-label="Provide feedback about this page" tabindex="0">Was this page helpful?</a>
```

Comment:
```suggestion
  <button type="button" class="usa-button usa-button--outline" id="#touchpoints-modal-button" aria-label="Provide feedback about this page" tabindex="0">Was this page helpful?</button>
```

The link doesn't point anywhere, we should use a button instead.